libs/utils/utils.js

```diff
@@ -996,7 +996,7 @@ async function documentPostSectionLoading(config) {
 }
 
 async function processSection(section, config, isDoc) {
-  const inlineFrags = [...section.el.querySelectorAll('a[href*="#_inline"]')];
+  const inlineFrags = [...section.el.querySelectorAll('a[href*="#_inline"][class~="fragment"]')];
```

Comment:
This seems like it would be better and simpler...
```suggestion
  const inlineFrags = [...section.el.querySelectorAll('a.fragment[href*="#_inline"]')];
```